streaming/README.md

```diff
@@ -511,4 +511,100 @@ metrics, following the same pattern of the receiving side.
 
 For a practical example see [Monitoring ephemeral nodes](#monitoring-ephemeral-nodes).
 
+## Troubleshooting streaming connections
+
+Sometimes when you are configuring master and slave nodes you can have some errors in your log files, in this section it
```

Comment:
This section describes the most common issues you may encounter in a connection between a slave and a master.

Reply:
It is done.

Reply:
Not "it is described the most common errors". Just copy paste what I wrote.

Reply:
Considering that you approved this, I am considering that you saw I had copied your text to the documentation! :)